docs/README-agent-install.md

```diff
@@ -27,6 +27,8 @@ This document describes Puppet agent installation and setup on Cisco Nexus switc
 
 See [References](#references) for alternative installation methods.
 
+**NOTE:** When using `OAC` and `Native Bash` hosting options, agentless is the recommended mode of operation for the Cisco Nexus module. Agent installations are not recommended because the agent won't be supported beyond Puppet Enterprise 2018.1. For more information, see [Puppet Enterprise support lifecycle](https://puppet.com/misc/puppet-enterprise-lifecycle).
```

Comment:
I would recommend the following wording.

The puppet agent will not be supported for the OAC or Native Bash hosting options beyond Puppet Enterprise 2018.1.  The agentless workflow is now recommended for managing the Cisco NX-OS devices.  Agent based workflows are still supported in the NX-OS Guestshell hosting environment.

Reply:
@mikewiebe Thanks! Updated.